CHANGELOG.asciidoc

```diff
@@ -26,6 +26,7 @@ image::https://raw.githubusercontent.com/apache/tinkerpop/master/docs/static/ima
 TinkerPop 3.3.0 (Release Date: NOT OFFICIALLY RELEASED YET)
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
+* `UnfoldStep` now supports unfolding of arrays. (*breaking*)
```

Comment:
I don't really like the use of "breaking" in the CHANGELOG for our manual entries. The JIRA ticket will have that when we generate that list on release.  Also, we've not done that consistently from the start of CHANGELOG. If something is "breaking" the better way to call attention to it is in the upgrade docs where we can better describe and demonstrate the change for users rather than a single one-liner in CHANGELOG.